.github/actions/test-go-tfe/action.yml

```diff
@@ -108,7 +109,7 @@ runs:
         GO111MODULE: "on"
         ENABLE_TFE: ${{ inputs.enterprise }}
       run: |
-        gotestsum --junitfile summary.xml --format short-verbose -- -parallel=1 -timeout=59m -coverprofile cover.out -run "${{ steps.test_split.outputs.run }}"
```

Comment:
I had a thought, take it or leave it - we might be able to introduce a new env var here that would cause go-tfe tests to fail if they tried to start a run without the `_RunDependent` convention.

Reply:
Mind expanding a bit on what you mean by the ENV var here 😅 

Though reading this gave me another idea that I think might be along the lines of what you're thinking.  [In this commit](https://github.com/hashicorp/go-tfe/pull/1237/commits/6a24a8dca6cab99035c7c95fe64aed7e622e34e6) I added guards to the helper funcs that create runs in order to enforce the naming convention. There were three basic ones that compose the rest.